text/0232-maturity-of-otel.md

```diff
@@ -0,0 +1,69 @@
+# Definition of maturity levels to be uniformly used by OpenTelemetry SIGs
+
+On 08 Mar 2023, the OpenTelemetry GC and TC held an OpenTelemetry Leadership summit, discussing various topics. One of the themes we discussed was establishing standard rules for describing the maturity of the OpenTelemetry project. This OTEP summarizes what was discussed there and is intended to have the wider community provide feedback.
+
+This OTEP builds on what was previously communicated by the project, especially the [Versioning and stability for OpenTelemetry clients](https://opentelemetry.io/docs/reference/specification/versioning-and-stability).
+
+The Collector's [stability levels](https://github.com/open-telemetry/opentelemetry-collector#stability-levels) inspired the maturity levels.
+
+## Motivation
+
+Quite often, the community is faced with the question of the quality and maturity expectations of its diverse set of components. This OTEP aims to bring clarity by establishing a framework to communicate the maturity for components delivered by SIGs in the name of the project. As the OpenTelemetry project comprises a multitude of SIGs, and each SIG has several components of varying quality, having this framework will help set the right expectations for OpenTelemetry users using a unified nomenclature.
```

Comment:
Can we please define what "component" means in this context? We've OTel level components (Collector, instrumentation, etc.) and components within certain projects (Collector components, for instance).

Reply:
I changed the second occurence of "components" so that it now reads:

> maturity for SIG deliverables in the name of the project

Reply:
> maturity for SIG deliverables (or components) in the name of the project

Something combining both? I think using `components` is still useful if defined properly.

Reply:
Sounds good, I replaced "or" with "and".